app/javascript/src/LoginPage/utils/formValidation.js

```diff
@@ -0,0 +1,64 @@
+import validate from 'validate.js'
+
+const constraintsTypes = {
+  text: {
+    presence: true,
+    length: {minimum: 1}
+  },
+  email: {
+    presence: true,
+    email: true,
+    length: {minimum: 1}
+  },
+  password: {
+    presence: true,
+    length: {minimum: 6}
```

Comment:
Confirmed, minimum password length is 6.
Additionally, it will be needed to show error messages from server validation (e.g, password too short, username already token, etc.), but I'll make a new PR for that, because that implies updating some fields in the UI in all forms